DataStructures/BitArray/TestBitArray.cs

```diff
@@ -0,0 +1,377 @@
+﻿/**
+ * Author: Christian Bender
+ * Class: TestBitArray
+ * 
+ * This class contains some tests for the class BitArray.
+ * 
+ * */
+
+
+using NUnit.Framework;
+using System;
+
+namespace DataStructures.BitArray
+{
+	[TestFixture ()]
+	public class TestBitArray
+	{
+		/*
+		 * Test for methods Compile(string) and ToString()
+		 * */
+		[Test ()]
+		[ExpectedException ("System.Exception")]
+		public void TestCompileToString ()
+		{
+			BitArray testObj = new BitArray (5);
+
+			testObj.Compile ("00100");
+			Assert.AreEqual (testObj.ToString (), "00100");
+
+			testObj.Compile ("01101");
+			Assert.AreEqual (testObj.ToString (), "01101");
+
+			testObj.Compile ("100"); // appropriate scaling
+			Assert.AreEqual (testObj.ToString (), "00100");
+
+			// throws exception
+			testObj.Compile ("klgml");
+			testObj.Compile ("010111111");
+		}
+
+
+		/*
+		 * Test constructor (sequence : string)  and constructor (bits : bool[])
+		 * */
+		[Test ()]
+		[ExpectedException ("System.Exception")]
+		public void TestConstructor ()
+		{
+			BitArray testObj1 = new BitArray ("00100");
+			BitArray testObj2 = new BitArray ("000120"); // throws exception
+			BitArray testObj3 = new BitArray (new bool[3] { true, false, true });
+
+			Assert.AreEqual (testObj1.ToInt64 (), 4);
+			Assert.AreEqual (testObj1.ToInt64 (), 5);
+
+
+
```

Comment:
Unwanted empty lines between #54 and #58.

Reply:
I have went through the entire file and found multiple instances like this (unnecessary empty lines), so I won't add comments regarding all of them, but you can treat this comment as though it refers to all of them. It would be really awesome if you clean up the file from unnecessary empty lines.